index.js

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable no-console */
+function validateLineup(lineup) {
+  return validateSalary(lineup) && validateTeams(lineup) && validateGames(lineup) && validatePositions(lineup)
+}
+// check if salary total > $45,000)
+function validateSalary(lineup) {
+  let checkCorrectSalary = lineup.map((correctsalary) => correctsalary.salary)
+
+  let combinedSalary = checkCorrectSalary.reduce((acc, num) => acc + num, 0)
```

Comment:
why map first then reduce?

can reduce without mapping where instead of  num being a salary number, it will be the player and you can get salary from inside the player object.